src/play-land.test.js

```diff
@@ -0,0 +1,96 @@
+import { Client } from "boardgame.io/client";
+import { Local } from "boardgame.io/multiplayer";
+import { mtg } from "./mtg";
+import { LAND } from "./permanents";
+
+const setup = playerStates => {
```

Comment:
I prefer the non-tuple version of this kind of API to avoid renaming of variables. I find that this makes the code easier to follow a lot of the time:

```suggestion
const setup = ({ playerStates }) => {
```

Reply:
agree